lib/generators/boring/vcr/install/install_generator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Boring
+  module Vcr
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds VCR to the application"
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :test_framework,
+                   type: :string,
+                   alias: "-f",
+                   default: "rspec",
+                   enum: %w[rspec minitest],
+                   desc: "Tell us which test framework you are using. Default to rspec"
+
+      class_option :stubbing_libraries,
+                   type: :array,
+                   alias: "-sl",
+                   default: ['webmock'],
+                   enum: %w[webmock typhoeus faraday excon],
+                   desc: "Tell us stubbing library you want to use seprated by space. Default to webmock"
+
+      def add_vcr_gem
+        say "Adding VCR gems to Gemfile", :green
+        # TODO: Use check_and_install_gem method when it is available
```

Comment:
Let's create an issue in our project and we can take a look at it instead of leaving TODO like this

Reply:
Yes maybe just insert the bundle install inside that helper as well so we don't have to run it separately for each gem?

For issues like TODO, I mean in our Truemark's Boring Generator Github